saw-core-what4/src/Verifier/SAW/Simulator/What4.hs

```diff
@@ -1060,7 +1057,9 @@ w4EvalAny ::
   SAWCoreState n ->
   SharedContext ->
   Map Ident (SValue (B.ExprBuilder n st fs)) -> Set VarIndex -> Term ->
-  IO ([String], ([Maybe (Labeler (B.ExprBuilder n st fs))], SValue (B.ExprBuilder n st fs)))
+  IO ([String],
+      [SValue (B.ExprBuilder n st fs)],
+      ([Maybe (Labeler (B.ExprBuilder n st fs))], SValue (B.ExprBuilder n st fs)))
```

Comment:
nitpick: is there a good reason to retain this nested tuple structure?

Reply:
Probably not. I'll flatten it.